src/components/inputs/promocode-input.js

```diff
@@ -46,7 +46,7 @@ const PromocodeInput = ({summitId, error, value, onChange, id, multi, ...rest})
             callback(newOptions);
         };
 
-        queryPromocodes(summitId, input, translateOptions);
+        queryPromocodes(summitId, input, translateOptions, DEFAULT_PAGE_SIZE, extraFilters);
```

Comment:
allow to define the page size using a prop and if not defined use then DEFAULT_PAGE_SIZE